packages/router/utils.ts

```diff
@@ -372,9 +372,13 @@ function flattenRoutes<
   parentsMeta: RouteMeta<RouteObjectType>[] = [],
   parentPath = ""
 ): RouteBranch<RouteObjectType>[] {
-  routes.forEach((route, index) => {
+  let flattenRoute = (
+    route: RouteObjectType,
+    index: number,
+    relativePath?: string
+  ) => {
     let meta: RouteMeta<RouteObjectType> = {
-      relativePath: route.path || "",
+      relativePath: relativePath ?? (route.path || ""),
```

Comment:
@brophdawg11 do we transpile RR package to browser-compatible JS? I assume yes, but just 2x checking to make sure `??` doesn't cause issues.

Reply:
Let's remove for now since that could be a problem in remix - let's revisit that as part of https://github.com/remix-run/remix/discussions/4769?